service/src/test/java/tds/config/web/endpoints/ConfigControllerIntegrationTests.java

```diff
@@ -30,6 +32,7 @@
 
 @RunWith(SpringRunner.class)
 @WebMvcTest(ConfigController.class)
+@Import(ExceptionAdvice.class)
```

Comment:
The integration tests will need to import Exception advice.  This is due to how Spring WebMvcTest only loads certain things into the application context.  I feel like Spring should pull in the advice without us having to import it and it is a bug in their testing harness.